ui/segmented-bar.reel/segmented-bar.js

```diff
@@ -0,0 +1,259 @@
+/**
+ * @module montage/ui/segmented-bar.reel
+ * @requires montage/ui/base/abstract-button.AbstractButton
+ * @requires montage/core/tree-controller.TreeController
+ */
+var AbstractButton = require("ui/base/abstract-button").AbstractButton;
+
+/**
+ * @class SegmentedBar
+ * @classdesc
+ * A segmented bar.
+ * Reference design: OS X storage space visualization found in iTunes or About This Mac
+ * Inheriting from AbstractButton because SegmentedBar needs its PressComposer behavior
+ *
+ * @extends AbstractButton
+ *
+ * @see {@link https://developer.mozilla.org/en-US/docs/Web/Guide/CSS/Flexible_boxes}
+ * for FlexBox usage
+ */
+exports.SegmentedBar = AbstractButton.specialize( /** @lends SegmentedBar.prototype # */ {
```

Comment:
> line 110: I think this is wrong... it seems that this will put in the event the "content" object of the selected segment, which is simply a cover for the segment's length and label properties when the segment is a Segment, and may be nothing or something entirely different when the segment is a custom class. What should be in the event is the Repetition:iterator.object corresponding to the segment clicked on.

Are you sure you're referring to the right line w/ the `@todo`? I'm confused.